core/arch/proc.h

```diff
@@ -491,14 +492,12 @@ proc_num_simd_saved(void);
 
 DR_API
 /**
- * Returns the number of SIMD registers preserved for an context switch. The number
- * returned here depends only on the processor and OS feature bits and reflects the
- * number of active SIMD registers, regardless of whether DynamoRIO may be optimizing
- * context switches or not.
+ * Returns the number of SIMD registers. The number returned here depends on the
```

Comment:
Since MMX registers are also "SIMD" this may need clarification for x86.  Ditto for proc_num_simd_saved().

Reply:
I've added "For x86 this only includes the vector registers (xmm/ymm/zmm)."

Reply:
I would think mmx could also be called "vector registers".  I don't know if there's a good way to distinguish beyond saying "recent" or "SSE+" or something.  Plus, maybe it's better to exclude mmx so "aamm" or whatever the 1024-bit ones will be covered without us having to edit "xmm/ymm/zmm".